### ISSUE TITLE
Security Audits

### ISSUE THREAD

**Issue by xaur** [xaur/decred-issues#55]
Let third party company (or multiple companies) audit the core pieces of software: dcrd, dcrwallet, dcrstakepool and maybe Decrediton and mobile wallets.

This can evolve into a 2-stage RFP-like Politeia vote, first vote to determine if stakeholders want to fund such audit(s) and second vote to choose a company.

Alternatively (or additionally), If there are security researchers who audit projects for fun, fame and optional rewards, that's cool too.

**Comment by degeri** [comment]
Guess I am already doing this albeit in a more informal way. We could also bring on board  quality/consistent security researchers who take part in the bug bounty program and show a real interest in the project.

All this on top of having a company do a proper audit will help alot with security.

**Comment by xaur** [comment]
Good point, the bug bounty program will help to attract security people that add to our collective knowledge of which firms to hire (if any).
